app/bundles/FormBundle/Model/SubmissionModel.php

```diff
@@ -1130,4 +1130,86 @@ protected function validateActionCallbacks(SubmissionEvent $event, &$validationE
 
         unset($args, $actions, $availableActions);
     }
+
+    /**
+     * Return Forms with results to contacts tab.
+     *
+     * @param int|null $leadId
+     * @param bool     $inContactTab
+     *
+     * @return array
+     */
+    public function getFormsWithResults($leadId = null, $inContactTab = false)
```

Comment:
Looking at this, can we make the $leadId parameter required? Because the rest of the code expects it's configured. Update the annotation too please.